outputs/client.go

```diff
@@ -89,6 +90,10 @@ func (c *Client) Post(payload interface{}) error {
 
 	client := &http.Client{}
 
+	if c.Config.CheckCert == false {
+		client.Transport = &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}
```

Comment:
I would clone http.DefaultTransport instead. It has tons of interesting default options, like taking proxy settings from envvars